File: Backend/DenariiMobile/models/wallet_details.py
from django.db import models


class WalletDetails(models.Model):
    user_id = models.TextField()
    wallet_name = models.TextField()
    wallet_password = models.TextField()
    seed = models.TextField()
    balance = models.FloatField()
    wallet_address = models.TextField()

    def __str__(self):
        return self.wallet_name

File: Ios/denarii/denarii/WalletDetails.swift
//
//  WalletDetails.swift
//  denarii
//
//  Created by Andrew Katson on 5/28/23.
//

import Foundation

struct WalletDetails : Codable {
    var balance: Double = 0.0
    var seed: String = ""
    var userIdentifier: Int = 0
    var walletName: String = ""
    var walletPassword: String = ""
    var walletAddress: String = ""
}
